app/src/main/java/com/android/userinfo/Constants.kt

```diff
@@ -0,0 +1,12 @@
+package com.android.userinfo
+
+const val USERNAME = "username"
+const val EMAIL = "email"
+const val PHONE = "phone"
+const val PINCODE = "pincode"
+const val ADDRESS = "address"
+
+
+const val TOAST_EMAIL = "Email should have @ and .com or co.in"
```

Comment:
I do not think this should be in constant. It should be in string resources. Because if some one using different language( say telugu), we have to show the toast message in that language. String resources helps us with that ( localization)